util/cron/test-perf.gpu-ex-rocm-54.bash

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+#
+# Run GPU performance tests
+
+CWD=$(cd $(dirname ${BASH_SOURCE[0]}) ; pwd)
+source $CWD/common-native-gpu.bash
+source $CWD/common-hpe-cray-ex.bash
+
+export CHPL_COMM=none
+export CHPL_LLVM=system
+unset CHPL_LLVM_CONFIG  # we need this to avoid warnings
+export CHPL_LOCALE_MODEL=gpu
+export CHPL_LAUNCHER_PARTITION=bardpeak  # bardpeak is the default queue
+export CHPL_GPU=amd  # also detected by default
+export CHPL_GPU_ARCH=gfx90a
+export CHPL_NIGHTLY_TEST_CONFIG_NAME="perf.gpu-ex-rocm-54"
+
+export CHPL_TEST_PERF_CONFIG_NAME="1-node-mi60"
```

Comment:
What should the name of the config name be here? I don't know what model of rocm GPUs are on this system and rocm-smi isn't useful on the login node.

Reply:
`1-node-mi250x`

https://www.amd.com/en/products/accelerators/instinct/mi200/mi250x.html